doc/filters.md

```diff
@@ -178,6 +179,9 @@ mainPool:
 | mirrorPool     | [proxy.PoolSpec](#proxyPoolSpec)               | Definition a mirror pool, requests are sent to this pool simultaneously when they are sent to candidate pools or main pool                                                                                                                                                                                          | No       |
 | failureCodes   | []int                                          | HTTP status codes need to be handled as failure                                                                                                                                                                                                                                                                     | No       |
 | compression    | [proxy.CompressionSpec](#proxyCompressionSpec) | Response compression options                                                                                                                                                                                                                                                                                        | No       |
+| mtls           | [proxy.MTLS](#proxymtls)            | MTLS configuration | No |
```

Comment:
```suggestion
| mtls           | [proxy.MTLS](#proxymtls)            | mTLS configuration | No |
```

Please use https://www.tablesgenerator.com/markdown_tables# to generate the table